src/one_dragon/utils/cv2_utils.py

```
@@ -931,3 +931,148 @@ def find_character_avatar_center_with_offset(img: MatLike, area_offset: Tuple[in
+    hsv_roi = cv2.cvtColor(roi, cv2.COLOR_RGB2HSV)
+
+    # S、V通道保持原有的线性计算
+    s_min, v_min = np.min(hsv_roi[:, :, 1:], axis=(0, 1))
+    s_max, v_max = np.max(hsv_roi[:, :, 1:], axis=(0, 1))
+
+    # 使用双区间比较算法计算H通道的中心值和差值
+    h_values = hsv_roi[:, :, 0].flatten()
+    h_min = int(np.min(h_values))
+    h_max = int(np.max(h_values))
+
+    # 计算线��区间长度：[min, max]
```

Ruff (0.14.2)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)